docs/sources/mimir/configure/configure-spread-minimizing-tokens/index.md

```diff
@@ -15,9 +15,14 @@ Using this guide, you can configure Mimir's ingesters to use the _spread-minimiz
 
 The ingester time series replication should be [configured with enabled zone-awareness](https://grafana.com/docs/mimir/latest/configure/configure-zone-aware-replication/#configuring-ingester-time-series-replication).
 
-{{% admonition type="note" %}}Spread-mimizing tokens are recommended if [shuffle sharding](https://grafana.com/docs/mimir/latest/configure/configure-shuffle-sharding/#ingesters-shuffle-sharding) is disabled in your ingesters, or, if shuffle-sharding is enabled, but most of the tenants of your system use all available ingesters.
+{{% admonition type="note" %}}Spread-mimizing tokens are recommended if [shuffle-sharding](https://grafana.com/docs/mimir/latest/configure/configure-shuffle-sharding/#ingesters-shuffle-sharding) is disabled on the [write path](https://grafana.com/docs/mimir/latest/configure/configure-shuffle-sharding/#ingesters-write-path) of your ingesters, or, if it is enabled, but most of the tenants of your system use all available ingesters.
```

Comment:
I'm not sure what the phrase "all available ingesters" refers to. This sounds vague to me- can we just remove the word "all" here?

Reply:
Shuffle-sharding is a feature where you reduce the ingester count handling a specific tenant's series to a subset of ingesters - but you may not use it for all tenants, only some. So the phrasing "most tenants of your systems using all available ingesters" does make sense in that context since there is a possibility for most tenants of your system to /not/ use all available ingesters.

Reply:
Thanks for clarifying! This makes sense to me now :)